app/backend/approaches/chatreadretrieveread.py

```diff
@@ -58,7 +58,7 @@ def system_message_chat_conversation(self):
         return """Assistant helps the company employees with their healthcare plan questions, and questions about the employee handbook. Be brief in your answers.
         Answer ONLY with the facts listed in the list of sources below. If there isn't enough information below, say you don't know. Do not generate answers that don't use the sources below. If asking a clarifying question to the user would help, ask the question.
         If the question is not in English, answer in the language used in the question.
-        Each source has a name followed by colon and the actual information, always include the source name for each fact you use in the response. Use square brackets to reference the source, for example [info1.txt]. Don't combine sources, list each source separately, for example [info1.txt][info2.pdf].
+        {sources_reference_content}
```

Comment:
We've considered moving the prompts into Jinja files. I'm wondering if we should make that change first, as this could read a bit nicer, like
```
{% if should_reference_sources $}
Each source...
{% endif %}
```

Our current method of storing the templates inside multi-line strings is not easy to work with, so I've been hoping to move them at least into separate files, and just hadn't decided about .txt versus Jinja versus prompty files. Jinja is a happy medium. That could be done by you or I in a separate PR. Thoughts?

Reply:
Agreed. I'll create a separate PR for this first.

Reply:
Implemented in #2164